manifests/base/monitoring/kustomization-no-crd.yaml

```
@@ -0,0 +1,18 @@
+
+resources:
+  - network-policy.yaml
+  - metrics-service.yaml
+  - dashboards  # Include Perses dashboard ConfigMaps for training metrics
+
+# Common labels applied to all resources
+commonLabels:
+  app: training-operator
+  app.kubernetes.io/name: training-operator
+  app.kubernetes.io/part-of: kubeflow
+  monitoring.opendatahub.io/component: "true"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)